app/src/main/java/com/github/codetanzania/adapter/IssueListPagerAdapter.java

```diff
@@ -46,6 +46,9 @@ public IssueListPagerAdapter(Context ctx, FragmentManager fm, ArrayList<ServiceR
 
         ServiceRequestsUtil.sort(all);
         for (ServiceRequest request : all) {
+            if (request == null) {
```

Comment:
This check filters null items from being added in either `open` or `closed` issues. Potentially, a null item might still be present in the `all` items list.